cluster-agent/values.yaml

```diff
@@ -34,11 +40,8 @@ agent:
   snapshotSyncInterval: 15
   agentServerPort: 8989
 
-
-
 serviceAccount:
   create: true
   name: appdynamics-operator
 
-metrics-server:
-  install: true
+existingSecret:
```

Comment:
Added existing secret so that user can use a pre-created secret to pass sensitive information.